### PR TITLE
ROCM:Unskip more tests for ROCM5.0

### PR DESCRIPTION
**Pull request by arindamroy-eng**
Re-enabling more tests which are working on ROCM5.0

Fixes #ISSUE_NUMBER

### CODE REVIEW

**Comment by arindamroy-eng** [comment]
@jeffdaily @jithunnair-amd

File: aten/src/ATen/cuda/CUDASparse.h [hunk 1]
```diff
@@ -34,8 +34,7 @@
 
 // BSR triangular solve functions were added in hipSPARSE 1.11.2 (ROCm 4.5.0)
 #if defined(CUDART_VERSION) ||                            \
-    (defined(USE_ROCM) && (hipsparseVersionMajor >= 1) && \
-     (hipsparseVersionMinor >= 11) && (hipsparseVersionPatch >= 2))
+      (defined(USE_ROCM) && ROCM_VERSION >= 40500 )
```
Comment (IvanYashchuk) [inline aten/src/ATen/cuda/CUDASparse.h:37]: Is it possible to install ROCm 4.5 with an older hipSPARSE version?

Comment (amathews-amd) [reply aten/src/ATen/cuda/CUDASparse.h:37]: hipSparse is part of ROCm and we dont support mix and match packages between different ROCm releases

**Comment by ezyang** [comment]
rocm tests are failing

**Comment by arindamroy-eng** [comment]
@pruthvistony

**Comment by arindamroy-eng** [comment]
@pruthvistony @amathews-amd @jithunnair-amd @jeffdaily 

I have rebased the whole patch and now its passing all tests.
Request to merge this soon.

**Review by ezyang — APPROVED** [review]

**Pull request closed without merging**